trackintel/analysis/location_identification.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+import pandas as pd
+import trackintel as ti
+
+
+def location_identifier(sps, recipe="FREQ", pre_filter=True, **pre_filter_kwargs):
```

Comment:
would be great if you could also add `location_identifier` and/or `pre_filter_locations` to the documentation.

Reply:
I have tried it, but I am not quite sure if it is correct. I just copied the code from the older docs.

Reply:
I checked and added a commit to fix the warnings in the docstring. You can test yourself by referring to our [guide ](https://github.com/mie-lab/trackintel/blob/master/CONTRIBUTING.md#documentation).